Naive greedy alg/scheduling/Scheduler.py

```diff
@@ -195,7 +195,7 @@ def _is_job_reconfigurable(self, job, av_servers, time):
             return False
 
         extra_srv_count = min(job.max_server_count - job.server_count, len(av_servers))
-        if self.param_enable:
+        if extra_srv_count > 0 and self.param_enable:
```

Comment:
The added condition should not be necessary as the function is called only if there are available servers and as is_reconfigurable already checks that there is room for at least one more server.